x-pack/libbeat/processors/add_cloudfoundry_metadata/add_cloudfoundry_metadata.go

```diff
@@ -0,0 +1,101 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build linux darwin windows
+
+package add_cloudfoundry_metadata
+
+import (
+	"time"
+
+	"github.com/elastic/beats/x-pack/libbeat/common/cloudfoundry"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/libbeat/processors"
+	jsprocessor "github.com/elastic/beats/libbeat/processors/script/javascript/module/processor"
+)
+
+const (
+	processorName = "add_cloudfoundry_metadata"
+)
+
+func init() {
+	processors.RegisterPlugin(processorName, New)
+	jsprocessor.RegisterPlugin("AddCloudFoundryMetadata", New)
```

Comment:
There is an ongoing conversation about stateful processors here: https://github.com/elastic/beats/pull/16349#issuecomment-590282511 cc @jsoriano. I wonder if we should skip adding it to the jsprocessor for now

Reply:
Will remove, don't think it will be used in that context anyway.